rdrf/rdrf/frontend/src/pages/proms_page/components/question.tsx

```diff
@@ -240,8 +240,25 @@ class Question extends React.Component<QuestionInterface, object> {
         );
     }
 
+    public valueWithinLimits(value, min, max) {
+        if (value === "") return true;  // allow to skip
+        const minValue = parseInt(min, 10);
+        const maxValue = parseInt(max, 10);
+        const inputValue = parseInt(value, 10);
+        if (!isNaN(minValue) && !isNaN(maxValue)) {  // if min and max are specified
+            if (minValue <= inputValue && inputValue <= maxValue) {
```

Comment:
return minValue <= inputValue && inputValue <= maxValue
is a bit cleaner

Reply:
changes made